tests/kernel/fake_run_ltp.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Extracts test commands from an LTP runfile and executes them on the guest.
+# Maintainer: Richard Palethorpe <rpalethorpe@suse.com>
+# More documentation is at the bottom
+
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi qw(is_serial_terminal :DEFAULT);
+use utils;
+use Time::HiRes qw(clock_gettime CLOCK_MONOTONIC);
+use File::Basename 'basename';
+use JSON;
+use serial_terminal;
+require bmwqemu;
+
+sub run {
+    my ($self) = @_;
+    my $test = $self->{context};
+
+    if (!$test) {
+        die "No context object";
+    }
+
+    script_run("echo $test->{name}: ");
```

Comment:
I did not get the motivation behind this "fake" module but why not run `assert_script_run` here?

Reply:
@okurz is it important to assert on echo? What could fail on it?